zad3/app/views/bestiary/novigrad_monsters.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>BestiaryController#novigrad_monsters</h1>
-<p>Find me in app/views/bestiary_controller/novigrad_monsters.html.erb</p>
+<table>
```

Comment:
Ten i dwa pozostałe widoki są bardzo podobne - można to uprościć używając [partiali](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) 😉